pkgs/os-specific/linux/keymash/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitea, kernel, kmod }:
+let
+  version = "0.1.1";
+in
+stdenv.mkDerivation {
+  pname = "keymash";
+  version = "${version}-${kernel.version}";
+
+  src = fetchFromGitea {
+    domain = "git.bsd.gay";
+    owner = "fef";
+    repo = "keymash";
+    rev = "v${version}";
+    hash = "sha256-sET3NKH8sS9tIvf68nLiNBVQ1so9bISim0Nxr4kd/0I=";
+  };
+
+  hardeningDisable = [ "pic" "format" ];
+  nativeBuildInputs = [ kmod ] ++ kernel.moduleBuildDependencies;
+
+  makeFlags = kernel.makeFlags ++ [
+    "KERNELRELEASE=${kernel.modDirVersion}"
+    "KERNEL_DIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+    "INSTALL_MOD_PATH=$(out)"
+  ];
+
+  meta = with lib; {
+    longDescription = ''
```

```suggestion
  meta = with lib; {
    description = "fill me";
    longDescription = ''
```

there should be a description for search.nixos.org